qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -14,3 +14,11 @@
                 - memory_protection_key:
                     cpu_model_flags += ",+pku,check"
                     flags = "pku ospke"
+        - amd:
+            vendor_id = 'amd'
```

Comment:
With https://github.com/avocado-framework/avocado-vt/pull/2225, we might use 'only HostCpuVendor.amd' as restriction.

Reply:
> With [avocado-framework/avocado-vt#2225](https://github.com/avocado-framework/avocado-vt/pull/2225), we might use 'only HostCpuVendor.amd' as restriction.

@huangyum  It can't work for me now, and I can't find the change even I Bootstrap.sh with "--upstream". We can change the format with intel together in one auto bug if it works well, how do you think?

Reply:
Sure, looks good to me.